Sources/SpeziOpenAI/MessagesView.swift

```diff
@@ -68,29 +69,35 @@ public struct MessagesView: View {
     ///   - chat: The chat messages that should be displayed.
     ///   - bottomPadding: A fixed bottom padding for the messages view.
     ///   - hideSystemMessages: If system messages should be hidden from the chat overview.
+    ///   - hideFunctionMessages: If function messages should be hidden from the chat overview.
     public init(
         _ chat: [Chat],
         bottomPadding: CGFloat = 0,
-        hideSystemMessages: Bool = true
+        hideSystemMessages: Bool = true,
+        hideFunctionMessages: Bool = true
     ) {
         self._chat = .constant(chat)
         self._bottomPadding = .constant(bottomPadding)
         self.hideSystemMessages = hideSystemMessages
+        self.hideFunctionMessages = hideFunctionMessages
     }
     
     
     /// - Parameters:
     ///   - chat: The chat messages that should be displayed.
     ///   - bottomPadding: A bottom padding for the messages view.
     ///   - hideSystemMessages: If system messages should be hidden from the chat overview.
+    ///   - hideFunctionMessages: If function messages should be hidden from the chat overview.
     public init(
         _ chat: Binding<[Chat]>,
         bottomPadding: Binding<CGFloat> = .constant(0),
-        hideSystemMessages: Bool = true
+        hideSystemMessages: Bool = true,
+        hideFunctionMessages: Bool = true
```

Comment:
I think it would be good to share a constant, that defines what messages should be hidden and is configured in a single place. If we follow the approach to use a `Set`, we can define something like this:
```swift
/// Displays the content of a `Chat` message in a message bubble
public struct MessageView: View {
    public enum Defaults {
        static let hideMessagesWithRoles: Set<Chat.Role> = [.system, .function]
    }
    
    // ...
```

You can then use the `MessageView.Defaults.hideMessagesWithRoles` constant in both the initializers of the `MessagesView` and `MessageView` to provide a good mechanism to define this once and share this across views.